nix/drun.nix

```diff
@@ -15,7 +15,7 @@ pkgs:
       # installed. You will normally not be bothered to perform
       # the command therein manually.
 
-      cargoSha256 = "sha256-YHLGj2pK9WvGxRW+T2sUUvA6UhNv+sc0m1NlR18sJAc=";
+      cargoSha256 = "";
```

Comment:
what the fresh hell is doing this?

Reply:
https://github.com/dfinity/motoko/blob/master/.github/workflows/update-hash.yml going rogue?

Reply:
I rebased and took that commit out. It didn't repeat itself. Let's monitor...